chat_bot_demo.py

```diff
@@ -323,14 +323,14 @@ def add_name_phone(output_list, address_book: AddressBook):
         try:
             record.add_phones(Phone(phone))
             print(address_book)
-            print(f'New phone {phone} of {name} is added')
+            print(f'New phone {phone} of {name.capitalize()} is added')
         except PhoneError:
```

Comment:
Maybe name can be with small letter

Reply:
I think that name must be with big letter!